docs/metrics.md

```diff
@@ -45,12 +51,42 @@ Number of current scheduling migrations. Type: Gauge.
 ### kubevirt_migrate_vmi_succeeded
 Number of migrations successfully executed. Type: Gauge.
 
+### kubevirt_virt_api_up_total
+The number of virt-api pods that are up. Type: Gauge.
+
 ### kubevirt_virt_controller_leading
 Indication for an operating virt-controller. Type: Gauge.
 
 ### kubevirt_virt_controller_ready
 Indication for a virt-controller that is ready to take the lead. Type: Gauge.
 
+### kubevirt_virt_controller_ready_total
+The number of virt-controller pods that are ready. Type: Gauge.
+
+### kubevirt_virt_controller_up_total
+The number of virt-controller pods that are up. Type: Gauge.
+
+### kubevirt_virt_handler_up_total
+The number of virt-handler pods that are up. Type: Gauge.
+
+### kubevirt_virt_operator_leading_total
+The number of virt-operator pods that are leading. Type: Gauge.
+
+### kubevirt_virt_operator_ready_total
+The number of virt-operator pods that are ready. Type: Gauge.
+
+### kubevirt_virt_operator_up_total
+The number of virt-operator pods that are up. Type: Gauge.
+
+### kubevirt_vm_container_free_memory_bytes_based_on_rss
+The current available memory of the VM containers based on the rss. Type: Gauge.
+
+### kubevirt_vm_container_free_memory_bytes_based_on_working_set_bytes
+The current available memory of the VM containers based on the working set. Type: Gauge.
+
+### kubevirt_vm_count
```

Comment:
[non-histogram and non-summary metrics should not have "_count" suffix](https://github.com/prometheus/client_golang/blob/v1.15.0/prometheus/testutil/promlint/promlint.go#L205).

I think the right term is number, see https://prometheus.io/docs/concepts/metric_types/#gauge.

Reply:
`_total`?

Reply:
Total is only for counter metrics, so kubvirt_number_of_vms or kubevirt_vm_number, @sradco WDYT?

Reply:
`kubvirt_number_of_vms` it is